CONTRIBUTING.md

```diff
@@ -14,3 +14,17 @@ The animations in Animate.css should follow a few key principles:
 - **Animations should be helpful.** They should be designed to guide users to a point of interest, ease natural reading order, or to communicate relationships between elements.
 - **Animations should feel familial.** Avoid introducing animations that feel out-of-place compared to the existing set.
 - **Animations should feel natural.** Animations should reflect, as much as is reasonable, motion that occurs in natural physics. Avoid extreme timing functions, and model animations on real-world events.
+
+## Code Styling
```

Comment:
It might be nice to include a note that mentions `.editorconfig` and `.prettierrc` help enforce some of these styling preferences!

Reply:
Good catch. I added them